python/valis/db/queries.py

```diff
@@ -57,17 +58,21 @@ def append_pipes(query: peewee.ModelSelect, table: str = 'stacked',
     if table not in {'stacked', 'flat'}:
         raise ValueError('table must be either "stacked" or "flat"')
 
+    # Run initial query as a temporary table.
+    temp = create_temporary_table(query, indices=['sdss_id'])
+
```

Comment:
How long do these temp tables stick around after creation?  We're using `append_pipes` in most, if not all, of the other endpoints.   Will we need to worry about the number and size of these tables hanging around the db?

Reply:
That's a good point. Temporary tables should only stick around while a connection or a transaction are open. But since `valis` never closes the connection that could be an issue. I think adding an `atomic` to the Peewee query to force a transaction should delete the temporary table after the query is done, but I'll check that. Worst case I think I can manually delete the table.